labgrid/driver/modbusdriver.py

```diff
@@ -26,21 +26,31 @@ def on_activate(self):
     def on_deactivate(self):
         self.client = None
 
+    def handle_error(self, action):
+        error_code = self.client.last_error()
+        if error_code == 7:
+            exc = self.client.last_except()
+            if exc not in [0, 5]:
```

Comment:
Some comments for the numeric values would be useful. Perhaps they are already available as Enums in pyModbusTCP?

Reply:
Done, We now additionally import the constants from pyModbusTCP and use instead of the magic values.